conjure-core/src/main/java/com/palantir/conjure/defs/validator/ValidImportsForSafety.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.defs.validator;
+
+import java.util.List;
+
+final class ValidImportsForSafety {
+
+    /*
+    Limiting to java.lang.Long for now, but would eventually want to support more external imports.
+     */
+    static final List<String> ALLOWED_IMPORTS = List.of("java.lang.Long");
```

Comment:
It looks like there are quite a few instances of people importing things from `com.palantir` packages: https://sourcegraph.palantir.build/search?q=context:global+external:+file:.yml+-repo:%5Egithub%5C.com/palantir/conjure&patternType=standard&sm=1&groupBy=repo Ignoring `com.palantir.logsafe.Safe` (which I think is the old way of marking safety?), I see lots of random types that might be useful to log like ResourceIdentifier, ObjectRid, User, GeotimeTrackGid, CosmosPlanGid, etc. I was imagining longs/primitives as a first step, and then later supporting safety annotations on most external imports, so static list for now, more complicated logic later.

Reply:
👍 That sounds good to me.

`ResourceIdentifier` is a funny one because conjure provides a first-class `rid` type using the same class! The other types are examples that we probably don't want to allow safety declarations on because the types themselves already declare their own safety -- it wouldn't be ideal if the external import declared a type as `safe` when the java class is annotated `@Unsafe`.